build/tasks.js

```diff
@@ -29,8 +29,20 @@ export default {
   },
 
   async runDev(args = []) {
+    const chokidar = require('chokidar');
```

Comment:
Do we really need chokidar? We can totally write this on our own. Maybe file an issue for a followup.